src/test/java/com/wikia/webdriver/elements/oasis/pages/InfoboxBuilderPage.java

```diff
@@ -90,10 +118,26 @@ public int countImages() {
     return images.size();
   }
 
+  public int countHeaders() {
+    return headers.size();
```

Comment:
I decided to leave it like this, because there's no specific element for which I can wait. I considered waiting for the first element, but it's cannot be used for Test Cases where there are none elements.